packages/worksheet/src/pages/WorksheetTemplate.js

```diff
@@ -9,7 +9,7 @@ import { Box, Button, HStack, Skeleton, Stack, VStack } from "native-base";
 import React from "react";
 import { useTranslation } from "react-i18next";
 import manifest from "../manifest.json";
-import "react-responsive-carousel/lib/styles/carousel.min.css"; // requires a loader
+import "react-responsive-carousel/lib/styles/carousel.min.css";
```

Comment:
why css laded here? can this be moved to package script?

Reply:
we only use here the carousel and carousel want import this css.